packages/ui/src/components/slider/Slider.tsx

```diff
@@ -0,0 +1,36 @@
+import type { ReactRangeSliderInputProps } from "react-range-slider-input";
+import RangeSlider from "react-range-slider-input";
+import "react-range-slider-input/dist/style.css";
+import { description, rangeSlider, rangeWrapper } from "./style.css";
+type SliderType = ReactRangeSliderInputProps & {
+  unit?: string;
+  width?: number;
+};
+const Slider = ({
+  value,
+  disabled,
+  onInput,
+  unit = "unit",
+  width = 200,
+  ...props
+}: SliderType) => {
+  return (
+    <div className={rangeWrapper} style={{ width: width }}>
+      <RangeSlider
+        disabled={disabled}
+        rangeSlideDisabled={disabled}
+        className={rangeSlider}
+        value={value}
+        onInput={onInput}
+        {...props}
+      />
+      <p className={description}>
+        {value?.[0]}
+        {unit} ~ {value?.[1]}
+        {unit}
```

Comment:
여기 코드가 가독성이 떨어져서 상수로 처리하면 좋을 거 같아
예를들어,
```suggestion
  const startValue = value?.[0] + unit;
  const endValue = value?.[1] + unit;
  return (
    <div className={rangeWrapper} style={{ width: width }}>
      <RangeSlider
        disabled={disabled}
        rangeSlideDisabled={disabled}
        className={rangeSlider}
        value={value}
        onInput={onInput}
        {...props}
      />
      <p className={description}>
        {startValue} ~ {endValue}
      </p>
    </div>
  );
```

Reply:
오 이런방법이~ 훨씬 낫네